### PR TITLE
Add OIDC description into operational layer doc

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/9773

### CODE REVIEW

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 1]
```diff
@@ -32,8 +32,27 @@ Giant Swarm SREs on this layer have root level SSH access to everything that per
 
 The Giant Swarm Control Plane consists mainly of services running inside the Control Plane Kubernetes cluster.
 
-Just like the former layer, this layer is accessed through VPN and bastion hosts. Giant Swarm SREs and operations personell have cluster admin access to the Control Plane Kubernetes API through a tunnel, which is again facilitated by SSO with MFA.
+Control Plane Kubernetes API network access is allowed only thorugh Giant Swarm VPN and customer VPN.
 
+Giant Swarm SREs and operations personell have cluster admin access to the Control Plane Kubernetes API through a tunnel, which is again facilitated by SSO with MFA.
```
Comment (axbarsan) [inline src/content/basics/giant-swarm-operational-layers/index.md:37]:
```suggestion
Giant Swarm SREs and operations personnel have cluster admin access to the Control Plane Kubernetes API through a tunnel, which is again facilitated by SSO with MFA.
```

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 2]
```diff
@@ -32,8 +32,27 @@ Giant Swarm SREs on this layer have root level SSH access to everything that per
...
+
+A customer has *tenant admin* and *view* access via OpenID Connect(OIDC), configured towards supported Identity Provider.
```
Comment (axbarsan) [inline src/content/basics/giant-swarm-operational-layers/index.md:39]:
```suggestion
A customer has *tenant admin* and *view* access via OpenID Connect (OIDC), configured towards the supported Identity Provider. 
```

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 3]
```diff
@@ -32,8 +32,27 @@ Giant Swarm SREs on this layer have root level SSH access to everything that per
...
+Giant Swarm SREs and operations personnel have cluster admin access to the Control Plane Kubernetes API through a tunnel, which is again facilitated by SSO with MFA.
+
+A customer has *tenant admin* and *view* access via OpenID Connect (OIDC), configured towards the supported Identity Provider. 
+
+#### Customer's Control Plane Kubernetes SSO authentication
+
+Every Control Plane Kubernetes has [dex](https://github.com/dexidp/dex) installed as OIDC issuer. Dex is configured with the selected by customer Identity Provider. List of supported providers can be found in the [dex github repository](https://github.com/dexidp/dex/tree/master/connector).
+Alongside with *dex* installed [dex-k8s-authenticator](https://github.com/mintel/dex-k8s-authenticator) - OIDC client for JWT token retrieval automation.
+
+#### Customer's Control Plane Kubernetes SSO authorization
```
Comment (oponder) [inline src/content/basics/giant-swarm-operational-layers/index.md:46]:
```suggestion
##### Authorization
```

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 4]
```diff
@@ -32,8 +32,27 @@ Giant Swarm SREs on this layer have root level SSH access to everything that per
...
-Just like the former layer, this layer is accessed through VPN and bastion hosts. Giant Swarm SREs and operations personell have cluster admin access to the Control Plane Kubernetes API through a tunnel, which is again facilitated by SSO with MFA.
+Control Plane Kubernetes API network access is allowed only thorugh Giant Swarm VPN and customer VPN.
 
+Giant Swarm SREs and operations personnel have cluster admin access to the Control Plane Kubernetes API through a tunnel, which is again facilitated by SSO with MFA.
+
+A customer has *tenant admin* and *view* access via OpenID Connect (OIDC), configured towards the supported Identity Provider. 
+
+#### Customer's Control Plane Kubernetes SSO authentication
+
+Every Control Plane Kubernetes has [dex](https://github.com/dexidp/dex) installed as OIDC issuer. Dex is configured with the selected by customer Identity Provider. List of supported providers can be found in the [dex github repository](https://github.com/dexidp/dex/tree/master/connector).
```
Comment (oponder) [inline src/content/basics/giant-swarm-operational-layers/index.md:43]:
```suggestion
The kubernetes API on every Control has [dex](https://github.com/dexidp/dex) installed as an OIDC issuer. Dex is configured with an Identity Provider chosen by the customer. A list of supported providers can be found in the [dex github repository](https://github.com/dexidp/dex/tree/master/connector).
```

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 5]
```diff
@@ -32,8 +32,27 @@ Giant Swarm SREs on this layer have root level SSH access to everything that per
...
+Alongside with *dex* installed [dex-k8s-authenticator](https://github.com/mintel/dex-k8s-authenticator) - OIDC client for JWT token retrieval automation.
```
Comment (oponder) [inline src/content/basics/giant-swarm-operational-layers/index.md:44]:
```suggestion
[dex-k8s-authenticator](https://github.com/mintel/dex-k8s-authenticator) is also installed, it is a web app that helps in the JWT token retrieval and kubectl configuration```

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 6]
```diff
@@ -32,8 +32,27 @@ Giant Swarm SREs on this layer have root level SSH access to everything that per
...
+
+#### Customer's Control Plane Kubernetes SSO authorization
+
+With the *jwt* token, received from its Identity Provider, customer can two levels of access:
```
Comment (oponder) [inline src/content/basics/giant-swarm-operational-layers/index.md:48]:
```suggestion
With a valid *jwt* token, received from your chosen Identity Provider, customers can have two levels of access:
```

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 7]
```diff
@@ -32,8 +32,27 @@ Giant Swarm SREs on this layer have root level SSH access to everything that per
...
+  - *view* 
+    - *get*/*list*/*watch* access to all resources in Control Plane Kubernetes, except `configmaps` and `secrets`.
```
Comment (oponder) [inline src/content/basics/giant-swarm-operational-layers/index.md:50]:
```suggestion
    - *get*/*list*/*watch* access to all resources in the Control Plane, except for `configmaps` and `secrets`. 
```

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 8]
```diff
@@ -32,8 +32,27 @@ Giant Swarm SREs on this layer have root level SSH access to everything that per
...
+
+Every Control Plane Kubernetes has [dex](https://github.com/dexidp/dex) installed as OIDC issuer. Dex is configured with the selected by customer Identity Provider. List of supported providers can be found in the [dex github repository](https://github.com/dexidp/dex/tree/master/connector).
+Alongside with *dex* installed [dex-k8s-authenticator](https://github.com/mintel/dex-k8s-authenticator) - OIDC client for JWT token retrieval automation.
+
+#### Customer's Control Plane Kubernetes SSO authorization
+
+With the *jwt* token, received from its Identity Provider, customer can two levels of access:
+  - *view* 
+    - *get*/*list*/*watch* access to all resources in Control Plane Kubernetes, except `configmaps` and `secrets`. 
+    - *get*/*list*/*watch* access to all resources (including `configmaps` and `secrets`) in tenant cluster namespace.
```
Comment (oponder) [inline src/content/basics/giant-swarm-operational-layers/index.md:51]:
```suggestion
    - *get*/*list*/*watch* access to all resources (including `configmaps` and `secrets`) in tenant cluster namespaces.
```

**Comment by corest** [comment]
I'm not only ok, but super happy with your help. Thanks @oponder :heart:

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 9]
```diff
@@ -32,8 +32,27 @@ Giant Swarm SREs on this layer have root level SSH access to everything that per
...
+Control Plane Kubernetes API network access is allowed only thorugh Giant Swarm VPN and customer VPN.
 
+Giant Swarm SREs and operations personnel have cluster admin access to the Control Plane Kubernetes API through a tunnel, which is again facilitated by SSO with MFA.
+
+A customer has *tenant admin* and *view* access via OpenID Connect (OIDC), configured towards the supported Identity Provider. 
+
+#### Control Plane Kubernetes API Access for Customers
+
+The kubernetes API on every Control has [dex](https://github.com/dexidp/dex) installed as an OIDC issuer. Dex is configured with an Identity Provider chosen by the customer. A list of supported providers can be found in the [dex github repository](https://github.com/dexidp/dex/tree/master/connector).
+[dex-k8s-authenticator](https://github.com/mintel/dex-k8s-authenticator) is also installed, it is a web app that helps in the JWT token retrieval and kubectl configuration```
```
Comment (axbarsan) [inline src/content/basics/giant-swarm-operational-layers/index.md:44]:
```suggestion
[dex-k8s-authenticator](https://github.com/mintel/dex-k8s-authenticator) is also installed, it is a web app that helps in the JWT token retrieval and kubectl configuration
```

**Pull request merged**